public/request/forum-topic-comment/preview.php

```diff
@@ -18,7 +20,8 @@
 
 return response()->json([
     'message' => __('legacy.success.ok'),
-    'postPreviewHtml' => Blade::render('<x-forum.post :parsedPostContent="$parsedPostContent" isPreview="true" />', [
-        'parsedPostContent' => Shortcode::render($normalizedInput),
+    'postPreviewHtml' => Blade::render('<x-forum.topic-comment :$variant>{{ $body }}</x-forum.topic-comment> ', [
+        'body' => Shortcode::render($normalizedInput),
+        'variant' => 'preview',
```

Comment:
preview is not rendering embedded links correctly
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/3bf0d29d-6a28-4b19-8b0a-a634a0e0717b)

Reply:
Thanks, fixed in latest.